rossum_api/api_client.py

```diff
@@ -168,11 +171,12 @@ async def fetch_all(
             f"{resource}", method, query_params, sideloads
         )
         # Fire async tasks to fetch the rest of the pages and start yielding results from page 1
+        last_page = self._determine_last_page(total_pages, max_pages)
```

Comment:
I would be even ok with inlining this with the power of the or. (Requesting 0 pages seems weird so fallbacking to total_pages in this case doesn't seem like too confusing). But up to you ;-) 

```python
last_page = min(total_pages, max_pages or total_pages)

Reply:
Yea, the `0` discouraged me. But I get the point about `0` being weird and it might make sense to fallback to total_pages. Will change

Reply:
applied :)